### DISCUSSION TITLE
[UX] Every time I want to create a new chat I click on plus and make a new folder

### DISCUSSION THREAD

**Discussion by remixer-dec** [open-webui/open-webui#7628] [General]
The plus icon makes more sense for new chat creation and being at the right corner makes it more likely to be clicked for people who are used to ChatGPT's layout. I think the easiest way to approach this is to replace the plus icon with another icon, maybe a folder icon.

**Comment by turbosnail11** [comment]
I came here to say the exact same thing.  I’m not even sure why I *need* to have folders, but I definitely never purposely try to create one.  In addition, the actual “New Chat” button looks more like a title and I *always* miss it the first time - it should be more distinct from everything else.

Ideally, it should say “New Chat +” as that’s what we’re used to and maybe be further down.  The New Folder option is used so infrequently that I should be in a sub menu somewhere.  After creating a folder I have no idea how to use it.  I can’t see how to move or start new chats in the newly created folder.

**Comment by remixer-dec** [comment]
seems to be fixed
